lib/util.js

```diff
@@ -791,25 +789,29 @@ function formatProperty(ctx, value, recurseTimes, visibleKeys, key, array) {
     if (array && key.match(/^\d+$/)) {
       return str;
     }
-    name = JSON.stringify('' + key);
+    name = JSON.stringify(`${key}`);
```

Comment:
@jasnell `JSON.stringify(typeOfString)` is not simply wrapping with `"`.

![image](https://cloud.githubusercontent.com/assets/1622515/16666602/18b2c858-44a6-11e6-8042-6a2b0f0bc30b.png)